docs/design/backfill-plugin.md

```diff
@@ -207,6 +207,49 @@ flowchart TD
 
 ** Name is the internal label for the node given by the operator, is only used for observability (logging and tracking) purposes.
 
+### gRPC Client Configuration Overrides
```

Comment:
Wrong place. Please drop these changes. In this case it's no necessary to go back and change the original design doc.